src/python/zquantum/core/utils.py

```diff
@@ -607,34 +606,6 @@ def scale_and_discretize(values: Iterable[float], total: int) -> List[int]:
     return result
 
 
-def hf_rdm(n_alpha: int, n_beta: int, n_orbitals: int) -> InteractionRDM:
```

Comment:
Why removing this?

Reply:
moved to `_utils.py` in `openfermion/zapata_utils` since it's more appropriate there